org.speed.dreams.yaml

```diff
@@ -0,0 +1,151 @@
+    sources:
+      - type: archive
+        url: http://downloads.sourceforge.net/sourceforge/giflib/giflib-5.1.4.tar.bz2
+        sha256: 2c171ced93c0e83bb09e6ccad8e3ba2b
+  
+  - name: freeglut
+    buildsystem: cmake-ninja
+    
+    sources:
+      - type: archive
+        url: https://downloads.sourceforge.net/freeglut/freeglut-3.0.0.tar.gz
+        sha256: fca52242f9344627a30f11487ee42002e6b0dacd
+        
+  - name: jasper
+    buildsystem: cmake-ninja
+    config-opts:
+    - -DCMAKE_INSTALL_LIBDIR=lib
+    - -DCMAKE_BUILD_TYPE=Release
+    - -DJAS_ENABLE_OPENGL=ON
+    - -DJAS_ENABLE_LIBJPEG=ON
+    - -DJAS_ENABLE_AUTOMATIC_DEPENDENCIES=OFF
+    - -DCMAKE_SKIP_RPATH=ON
+    - -DALLOW_IN_SOURCE_BUILD=ON
+    sources:
+      - type: archive
+        url: https://github.com/mdadams/jasper/archive/version-2.0.14.tar.gz
+        sha256: 85266eea728f8b14365db9eaf1edc7be4c348704e562bb05095b9a077cf1a97b
+        
+  - name: openscenegraph
+    buildsystem: cmake-ninja
+    config-opts:
+    - -DCMAKE_BUILD_TYPE=Release
+    sources:
+      - type: archive
+        url: https://github.com/openscenegraph/OpenSceneGraph/archive/OpenSceneGraph-3.4.1.tar.gz
+        sha256: 930eb46f05781a76883ec16c5f49cfb29a059421db131005d75bec4d78401fd5
+  
+  - name: libxmu
+    sources:
+      - type: archive
+        url: https://www.x.org/archive//individual/lib/libXmu-1.1.2.tar.bz2
+        sha256: 756edc7c383254eef8b4e1b733c3bf1dc061b523c9f9833ac7058378b8349d0b
+  
+  - name: plib
+    rm-configure: true
+    config-opts:
+      - CXXFLAGS=-fPIC
+    sources:
+      - type: archive
+        url: http://plib.sourceforge.net/dist/plib-1.8.5.tar.gz
+        sha1: c2cf7e3e1e58f7b63dae4bb21e4fa82c3e4d4cfc
+        
+      - type: script
+        dest-filename: autogen.sh
+        commands:
+          - autoreconf -vfi
+
+  - name: enet
+    sources:
+      - type: archive
+        url: http://enet.bespin.org/download/enet-1.3.13.tar.gz
+        sha256: 82501c0248330a3a6bcac907907760e59b18cb662ebb42fe2c91e1557f1bc8ab452bf3fe7477dc01fa7a6e3b6124e1eb8eac46382d5bbebb2394879f9568b5df
+  
+  - name: freealut
+    cleanup:
+      - /bin
+    sources:
+      - type: archive
+        url: https://github.com/vancegroup/freealut/archive/freealut_1_1_0.tar.gz
+        sha256: c3880d7dbb90c7db54e1d6b88aa0a34dd9e2d828fd389be0d2cbb2632b0885dd
+      - type: script
+        dest-filename: autogen.sh
+        commands:
+          - autoreconf -vfi
+          
+  - shared-modules/udev/udev-175.json
+  
+  - name: speed-dreams-2
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DSD_BINDIR=/app/bin
+      - -DSD_DATADIR=/app/share/speed-dreams-2
+      - -DSD_LIBDIR=/app/lib/speed-dreams-2
+      - -DOPTION_OFFICIAL_ONLY=ON
+    
+    sources:
+      - type: file
+        url: https://downloads.sourceforge.net/sourceforge/speed-dreams/speed-dreams-src-base-2.2.2-rc2-r6527.tar.xz
+        md5: 956bda0acbcb89eaa2ca742dbedc8941
```

Comment:
Use sha256 for checksums. It's de facto standard on Flathub.

Reply:
I change all checksum to sha256